discovery-provider/src/queries/get_trending_tracks.py

```diff
@@ -46,7 +46,13 @@ def get_trending_tracks(args):
     current_user_id = get_current_user_id(required=False)
 
     db = get_db_read_replica()
+
     time = args.get('time')
+    # Identity understands allTime as millennium.
+    # TODO: Change this in https://github.com/AudiusProject/audius-protocol/pull/768/files
```

Comment:
So is the plan for me to revert this back in my PR?

Reply:
Yeah, basically just destroy this line and add equivalent of making all time work there